config/initializers/datadog.rb

```diff
@@ -26,7 +26,7 @@
     c.tracing.instrument :redis, service_name: 'vets-api-redis'
     c.tracing.instrument :pg, service_name: 'vets-api-pg'
     c.tracing.instrument :http, service_name: 'vets-api-net-http'
-    c.tracing.log_injection.enabled = false
+    c.tracing.log_injection = false
```

Comment:
I don't think it liked the `=`. Does the format need to be like the other variables?
The error said: 
```
NoMethodError: undefined method `enabled=' for true (NoMethodError)
```

Reply:
`c.appsec.enabled = true`

Reply:
I don't think log_injection will work but this will be removed if it breaks dev again.

Reply:
ah, okay 👍 
I'll approve once tests pass